ga4gh/frontend.py

```diff
@@ -9,15 +9,19 @@
 
 import os
 import traceback
+import datetime
 
 import flask
 import flask.ext.api as api
```

Comment:
Don't think we're using this now? I use [flake8](https://pypi.python.org/pypi/flake8) in my editor, it catches this sort of thing nicely.